ci/build.sh

```diff
@@ -192,13 +194,14 @@ function generate_overrides() {
   echo "Generating setup job yaml from helm chart..."
   echo "# This file is auto-generated." > deploy/kubernetes/setup-sumologic.yaml.tmpl
 
-  with_files=$(find deploy/helm/sumologic/templates/setup/ -maxdepth 1 -iname "*.yaml" | sed 's#deploy/helm/sumologic/templates#-x templates#g' | sed 's/yaml/yaml \\/g')
+  # Get list of templates available for default values.yaml
+  with_files=$(find deploy/helm/sumologic/templates/setup -maxdepth 1 -iname "*.yaml" ! -path "*/setup-scc.yaml" | sed 's#deploy/helm/sumologic/templates#-s templates#g' | sed 's/yaml/yaml /g')
   # shellcheck disable=SC2086
   helm template deploy/helm/sumologic \
     ${with_files} \
     --namespace "\$NAMESPACE" \
-    --name collection \
-    --set dryRun=true >> deploy/kubernetes/setup-sumologic.yaml.tmpl \
+    --name-template collection \
+    --set dryRun=true >> deploy/kubernetes/setup-sumologic.yaml.tmpl 2>/dev/null \
```

Comment:
not specifically related to this PR but how does `>>` work here when used in the middle of parameter list?

Reply:
same like on the end